# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/basetypes/Accessible.h
# Commit: 5d76535b47fc
# Full Hash: 5d76535b47fced08e46753a0262211732fb26417
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P7: Move and rename GetPositionAndSizeInternal to Accessible. r=Jamie
#   
#   Now that the AccGroupInfo getter is in Accessible, the position and size
#   getter could go there too.
#   
# ==============================================================================

diff -r fca1ff7eabf4 -r 5d76535b47fc accessible/basetypes/Accessible.h
--- a/accessible/basetypes/Accessible.h	Tue Jan 04 21:01:36 2022 +0000
+++ b/accessible/basetypes/Accessible.h	Tue Jan 04 21:01:37 2022 +0000
@@ -319,6 +319,15 @@
    */
   virtual int32_t GetLevel(bool aFast) const;
 
+  /**
+   * Calculate position in group and group size ('posinset' and 'setsize') based
+   * on accessible hierarchy.
+   *
+   * @param  aPosInSet  [out] accessible position in the group
+   * @param  aSetSize   [out] the group size
+   */
+  virtual void GetPositionAndSetSize(int32_t* aPosInSet, int32_t* aSetSize);
+
  private:
   static const uint8_t kTypeBits = 6;
   static const uint8_t kGenericTypesBits = 18;